### DISCUSSION TITLE
Should I be concerned about the "has already been registered" Chrome warning?

### DISCUSSION THREAD

**Discussion by DaveSanders** [vuepress-theme-hope/vuepress-theme-hope#2706] [Q&A] [answered]
I've added the extended markdown plugin to a stock VuePress 2 app, and these warnings appeared:

```
"[Vue warn]: Component "CodeGroup" has already been registered in target app."
"[Vue warn]: Component "CodeGroupItem" has already been registered in target app."
```
I understand why these are happening, and as far as I can tell, they don't seem to have any effect, but for the purposes of trying to have a "clean" app, is there a configuration change needed to make these warnings go away?

**Comment by Mister-Hope** [comment, marked as answer]
Cannot be fixed on the plugin side. This is cause by legacy mode as we are using different syntax for code tabs.

You can safely omit it with default theme. To get raid of it, manually add `false` as second arg of `mdEnhancePlugin()`:


```ts
  plugins: [
    mdEnhancePlugin({ 
      // your options
     // ...
    }, false),
  ]
```

**Reply by DaveSanders** [reply]
Worked. Thank you!
